Binary files racon-v1.4.13-old/.git/modules/vendor/spoa/index and racon-v1.4.13/.git/modules/vendor/spoa/index differ
diff -ruN racon-v1.4.13-old/vendor/spoa/src/alignment_engine.cpp racon-v1.4.13/vendor/spoa/src/alignment_engine.cpp
--- racon-v1.4.13-old/vendor/spoa/src/alignment_engine.cpp	2021-09-14 14:24:53.358413371 +0200
+++ racon-v1.4.13/vendor/spoa/src/alignment_engine.cpp	2021-09-14 14:26:58.046107581 +0200
@@ -7,6 +7,7 @@
 #include <limits>
 #include <algorithm>
 #include <exception>
+#include <stdexcept>
 
 #include "sisd_alignment_engine.hpp"
 #include "simd_alignment_engine.hpp"
diff -ruN racon-v1.4.13-old/vendor/spoa/src/graph.cpp racon-v1.4.13/vendor/spoa/src/graph.cpp
--- racon-v1.4.13-old/vendor/spoa/src/graph.cpp	2021-09-14 14:24:53.358413371 +0200
+++ racon-v1.4.13/vendor/spoa/src/graph.cpp	2021-09-14 14:26:52.550165125 +0200
@@ -8,6 +8,7 @@
 #include <algorithm>
 #include <stack>
 #include <fstream>
+#include <stdexcept>
 
 #include "spoa/graph.hpp"
 
diff -ruN racon-v1.4.13-old/vendor/spoa/src/main.cpp racon-v1.4.13/vendor/spoa/src/main.cpp
--- racon-v1.4.13-old/vendor/spoa/src/main.cpp	2021-09-14 14:24:53.358413371 +0200
+++ racon-v1.4.13/vendor/spoa/src/main.cpp	2021-09-14 14:26:43.670258101 +0200
@@ -3,6 +3,7 @@
 #include <cstdint>
 #include <string>
 #include <iostream>
+#include <stdexcept>
 #include <exception>
 
 #include "sequence.hpp"
diff -ruN racon-v1.4.13-old/vendor/spoa/test/spoa_test.cpp racon-v1.4.13/vendor/spoa/test/spoa_test.cpp
--- racon-v1.4.13-old/vendor/spoa/test/spoa_test.cpp	2021-09-14 14:24:53.357413382 +0200
+++ racon-v1.4.13/vendor/spoa/test/spoa_test.cpp	2021-09-14 14:27:13.949941070 +0200
@@ -4,6 +4,8 @@
  * @brief Spoa unit test source file
  */
 
+#include <stdexcept>
+
 #include "spoa_test_config.h"
 
 #include "sequence.hpp"
